docs/api.md

```diff
@@ -0,0 +1,118 @@
+# HNPWA API
+
+**A fast, CDN delivered, aggregated Hacker News API**
+
+https://hnpwa.com/api/v0/news.json
+
+## Why?
+
+Everything in the official Hacker News API is either 
+[an item](https://hacker-news.firebaseio.com/v0/item/8863.json?print=pretty) or 
+[a list of items](https://hacker-news.firebaseio.com/v0/topstories.json?print=pretty).
+
+This structure is not ideal for building fast loading Progressive Web Apps since it will require multiple network requests
+to get the needed payload for page render. 
+
+The HNPWA solves this problem by aggregating each item in the id list into item feeds.
+
+## Item feeds
+
+There are five item feeds.
+
+| Name | URL |
+| --- | --- |
+| News| https://hnpwa.com/api/v0/news.json |
+| Newest | https://hnpwa.com/api/v0/newest.json |
+| Ask | https://hnpwa.com/api/v0/ask.json |
+| Show | https://hnpwa.com/api/v0/show.json |
+| Jobs | https://hnpwa.com/api/v0/jobs.json |
+
+### Schema
+
+Each item feed returns an array of `FeedItem`.
```

Comment:
Should the item feed also return the max? This would allow one to know the max number of pages available for the UI easily.

e.g. `{ max: 500, items: [FeedItem, ...] }`

Perhaps I'm missing where else this is provided.